config/metadata/twitter.ts

```diff
@@ -0,0 +1,10 @@
+const twitter = {
+  twitter: {
+    card: 'summary',
```

Comment:
just a quick change for @ecurrencyhodler 's suggestion

```suggestion
    card: 'summary_large_image',
```